stdlib/public/SDK/Dispatch/Data.swift

```diff
@@ -114,7 +114,7 @@ public struct DispatchData : RandomAccessCollection, _ObjectiveCBridgeable {
 	///
 	/// - parameter buffer: The buffer of bytes to append. The size is calculated from `SourceType` and `buffer.count`.
 	public mutating func append<SourceType>(_ buffer : UnsafeBufferPointer<SourceType>) {
-		self.append(UnsafePointer(buffer.baseAddress!), count: buffer.count * sizeof(SourceType.self))
+		self.append(UnsafePointer(buffer.baseAddress!), count: buffer.count * MemoryLayout<SourceType>.size)
```

Comment:
This code was changed in the `master` branch, would you mind rebasing?

Also, could you change `sizeof` to `strideof`?  Using `sizeof` here can cause bugs.